custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -92,6 +96,10 @@ def current_cover_tilt_position(self):
 
     async def async_set_cover_position_low_speed(self, **kwargs):
         """Move the cover to a specific position with a low speed."""
+        _LOGGER.warning(
+            "This service will be removed in Setptember 2021. Use instead the low speed switch."
```

Comment:
Let's remove the warning and keep the service as well. At the core PR we can figure out if people miss this :-).